README.md

```diff
@@ -44,6 +44,7 @@ yarn add react-recipes
 | 🥑 [`useOnClickOutside`](#-useOnClickOutside)   | -                                                     | (ref, callback)                                                                         | 
```

Comment:
Did you still want to change this emoji? We could do the orange here

Reply:
Let's review the emojis after I add more recipes (I have a list of ~10+)